vms/home/confirmed_email.html

```diff
@@ -0,0 +1,6 @@
+<html>
```

Comment:
Put this file in vms/home/templates/home. It is throwing an error. Also, try to include the navigation bar on this page too.

Reply:
Done, Also I have created a template for the page asking about email confirmation. Please review